tests/administration/users/new_user_validation.test.js

```diff
@@ -3,37 +3,19 @@ const common = require('../../common'),
       expect = require('chai').expect;
 
 describe('New User Validations', () => {
-  before(() => { 
-	common.login(browser, 'admin', 'admin', 'local' )
-	.then(() => common.clickSidebarTab(browser, 'Users', 'System Users'));
-  });
+  beforeEach(() => common.login(browser, 'admin', 'admin', 'local' )
+  	.then(() => common.clickSidebarTab(browser, 'Users', 'System Users')));
```

Comment:
Do we really need to log in and log out after every test? the tests that look for validation errors on a field within the form  don't really require that.

i guess this goes to for the previous and next test files... perhaps we should have a way to handle tests that require this and tests that don't (if possible)